pr-data.csv

```diff
@@ -1283,17 +1283,23 @@ https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalo
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.lazybinary.TestLazyBinarySerDe2.testLazyBinarySerDe,ID,,,
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/serde,org.apache.hadoop.hive.serde2.objectinspector.TestObjectInspectorUtils.testBucketHashGeneration,OD,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestObjectInspectorUtils.testObjectInspectorUtils,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestProtocolBuffersObjectInspectors.testProtocolBuffersObjectInspectors,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestReflectionObjectInspectors.testReflectionObjectInspectors,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestStandardObjectInspectors.testStandardUnionObjectInspector,ID,Accepted,https://github.com/apache/hive/pull/929,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftObjectInspectors,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftSetObjectInspector,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestObjectInspectorUtils.testObjectInspectorUtils,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestProtocolBuffersObjectInspectors.testProtocolBuffersObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestReflectionObjectInspectors.testReflectionObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestStandardObjectInspectors.testStandardUnionObjectInspector,ID,Accepted,https://github.com/apache/hive/pull/929,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftObjectInspectors,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.objectinspector.TestThriftObjectInspectors.testThriftSetObjectInspector,ID,,,
+https://github.com/apache/hive,ecdac363e042200d85bb8b894aa20d91a1642a18,serde,org.apache.hadoop.hive.serde2.TestStatsSerde.testLazyBinarySerDe,ID,,,
```

Comment:
Again, don't delete the info that's already there, but it's fine to add a new test.

Reply:
I will restore the file and update them again